packages/dds/map/src/directory.ts

```diff
@@ -649,7 +650,10 @@ export class SharedDirectory extends SharedObject<ISharedDirectoryEvents> implem
     protected reSubmitCore(content: any, localOpMetadata: unknown) {
         const message = content as IDirectoryOperation;
         const handler = this.messageHandlers.get(message.type);
-        handler.submit(message, localOpMetadata);
+
+        // eslint-disable-next-line max-len
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion, @typescript-eslint/no-unnecessary-type-assertion
+        handler!.submit(message, localOpMetadata);
```

Comment:
Would be nice to assert here rather than just a non-null assertion so that if we try the value type removal (thereby removing "act" type) then we can get a more descriptive error.
```typescript
assert(handler !== undefined, `Missing message handler for message type: ${message.type}`);
```

Reply:
done